setup.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import os
+from os.path import join
+from distutils.sysconfig import get_python_lib
+from setuptools import find_packages
+
+try:
+    from skbuild import setup
+    from skbuild.constants import CMAKE_INSTALL_DIR
+
+except ImportError:
+    print('scikit-build is required to build from source.', file=sys.stderr)
```

Comment:
why catch the ImportError? We don't do this for other dependencies

Reply:
yeah this dependency should be handled by pip. I forgot once or twice to install it so I put a warning..